test/liveValidation/swaggers/specification/rpsaas/resource-manager/Microsoft.Contoso/stable/2019-01-01/employee.json

```diff
@@ -192,7 +200,7 @@
           "200": {
             "description": "Operation details.",
             "schema": {
-              "$ref": "#/definitions/OperationsList"
+              "$ref": "#/definitions/NestedResource"
```

Comment:
Why to change this? Is this used in other tests?

Reply:
Since I add internal reference check before operation validate and will throw an exception , this json file need to fix the internal error, or there are 5 test cases would not passed